config/initializers/simple_form.rb

```diff
@@ -27,7 +27,7 @@
     b.optional :min_max
     b.optional :readonly
     b.use :label, class: 'text-bold'
-    b.use :hint,  wrap_with: { tag: 'div', class: 'usa-hint' }
+    b.use :hint,  wrap_with: { tag: 'label', class: 'usa-hints' }
```

Comment:
I'm a bit uncertain about using the `<label>` element here, since I would expect the hint would be provided _in addition to_ a label in most forms (this hint configuration applies to all forms).